resources/assets/js/components/FriendsList.vue

```diff
@@ -18,7 +18,16 @@
                         <img v-bind:src="friend.avatar_url" class="img-circle" height="60" />
                     </div>
                     <div class="friend__body">
-                        <h4><a href="http://twitter.com/{{ friend.username }}">@{{ friend.username }}</a></h4>
+                        <h4>
+                            <a href="http://twitter.com/{{ friend.username }}">
+                                <template v-if="friend.name != ''">
```

Comment:
You can just wrap this in `span` tags, rather than using `template`:

```html
<span v-if="friend.name != ''">
    {{ friend.name }} <small>(@{{ friend.username }})</small>
</span>
<span v-else>
    @{{ friend.username }}
</span>
```